ansible/deploy_docker.py

```diff
@@ -779,7 +779,9 @@ def wait_for_girder(client, ctn, maxWait=300):
         sys.stdout.flush()
     if not output:
         raise Exception('Girder never responded')
-    sys.stdout.write(' %s\n' % output['apiVersion'])
+    else:
```

Comment:
This doesn't have to be in an `else:` block, since it will only be reached if the exception wasn't raised.